node.gyp

```diff
@@ -207,7 +214,9 @@
         'src/udp_wrap.cc',
         'src/util.cc',
         'src/uv.cc',
+        'src/loader/module_wrap.cc',
         # headers to make for a more pleasant IDE experience
+        'src/loader/module_wrap.h',
```

Comment:
All of these should be alphabetical. `lib/internal/bootstrap_node.js',` is an exception for some historical reasons.